include/seqan3/alphabet/quality/composition.hpp

```diff
@@ -166,6 +167,16 @@ quality_composition(sequence_alphabet_type &&, quality_alphabet_type &&)
 
 } // namespace seqan3
 
+namespace seqan3::detail
+{
+
+//!\brief Since seqan3::quality_composition wraps a nucleotide alphabet it is also one.
+template <typename sequence_alphabet_type, typename quality_alphabet_type>
+struct is_nucleotide<quality_composition<sequence_alphabet_type, quality_alphabet_type>> : public std::true_type
+{};
+
+} // namespace seqan3::detail
+
 #ifndef NDEBUG
 // contains alphabet tests for this class:
 #include <seqan3/alphabet/quality/aliases.hpp>
```

Comment:
I would not this like that, I would the test cases here directly. It more easy to understand what you want to assert.

```c++
#include <seqan3/alphabet/nucleotide/dna4.hpp>
#include <seqan3/alphabet/quality/illumina18.hpp>

static_assert(seqan3::nucleotide_concept<seqan3::quality_composition<seqan3::dna4, seqan3::illumina18>>);
static_assert(seqan3::quality_concept<seqan3::quality_composition<seqan3::dna4, seqan3::illumina18>>);
```